client/src/components/Graph/Graph.tsx

```diff
@@ -173,11 +173,18 @@ class Graph extends React.Component<GraphProps, GraphState> {
   );
 
   computeHighlightFlags = memoize(
```

Comment:
Graph highlight logic now handles dict-encoded columns by mapping the label back to its numeric code. This lets the lasso + hotspots work for user annotations stored as categorical TileDB arrays.